sale_order_line_clone/readme/CONFIGURE.md

```diff
@@ -0,0 +1,4 @@
+\# Go to *Settings \> Users and Company \> Users* and edit users how can
```

Comment:
Please, consider replacing "how" with "that"

Reply:
Changed